cms/envs/common.py

```diff
@@ -836,11 +836,14 @@
     # Self-paced course configuration
     'openedx.core.djangoapps.self_paced',
 
-    # OAuth2 Provider
+    # django-oauth2-provider
     'provider',
     'provider.oauth2',
     'edx_oauth2_provider',
 
+    # django-oauth-toolkit
+    'oauth2_provider',
```

Comment:
Yeah.  Python as a language doesn't have any concept of an installable package, unfortunately.  That only exists at the setuptools/distutils/pip level.